packages/cfd/src/features/Containers/cfd-password-manager-modal/cfd-password-manager-tab-content.tsx

```diff
@@ -0,0 +1,194 @@
+            hideError();
+        }
+    }, []);
+
+    React.useEffect(() => {
+        handlePasswordErrorMessages(change_password_status, change_password_error as unknown as Error);
+    }, [change_password_error, change_password_status, handlePasswordErrorMessages]);
+
+    React.useEffect(() => {
+        handlePasswordErrorMessages(
+            change_investor_password_status,
+            change_investor_password_error as unknown as Error
+        );
+    }, [change_investor_password_error, change_investor_password_status, handlePasswordErrorMessages]);
+
+    const onSubmit = React.useCallback(
+        async (values: TPasswordManagerModalFormValues) => {
+            if (!selected_login) {
+                return;
+            }
+
+            if (values.password_type === PASSWORD_TYPE.INVESTOR) {
+                await changeInvestorPassword({
+                    account_id: selected_login,
+                    old_password: values.old_password,
+                    new_password: values.new_password,
+                    platform: CFD_PLATFORMS.MT5,
+                });
+            } else {
+                await changePassword({
+                    old_password: values.old_password,
+                    new_password: values.new_password,
+                    platform: CFD_PLATFORMS.MT5,
+                });
+            }
+        },
+        [changeInvestorPassword, changePassword, selected_login]
+    );
+
+    const updateAccountTabIndex = (index: number) => {
+        setActiveTabIndex(index);
+        onChangeActiveTabIndex(index);
+        setErrorMessageInvestor('');
+        setIsSubmitSuccessInvestor(false);
+    };
+
+    const trading_password_manager = (
+        <React.Fragment>
+            <DesktopWrapper>
+                <ThemedScrollbars height={container_height} is_bypassed={isMobile()} autohide={false}>
+                    <TradingPasswordManager
+                        toggleModal={toggleModal}
+                        platform={platform}
+                        email={email}
+                        account_group={account_group}
+                    />
+                </ThemedScrollbars>
+            </DesktopWrapper>
+            <MobileWrapper>
+                <Div100vhContainer className='cfd-password-manager__scroll-wrapper' height_offset='120px'>
+                    <TradingPasswordManager
+                        toggleModal={toggleModal}
+                        platform={platform}
+                        email={email}
+                        account_group={account_group}
+                    />
+                </Div100vhContainer>
+            </MobileWrapper>
+        </React.Fragment>
+    );
+
+    if (platform === CFD_PLATFORMS.DXTRADE) return trading_password_manager;
+
+    return (
+        <Tabs active_index={active_tab_index} onTabItemClick={updateAccountTabIndex} top>
+            <div
+                label={localize('{{platform}} password', {
```

Comment:
Please use `Localize`

Reply:
It will give type error as we are using `components` package and can not tweak it.
Or i have to override the type of label.